pkg/packagekit/updates.jsx

```diff
@@ -598,27 +624,25 @@ class OsUpdates extends React.Component {
 
     handleRefresh() {
         this.setState({state: "refreshing", loadPercent: null});
-        pkTransaction()
-            .done(transProxy =>  {
-                transProxy.addEventListener("ErrorCode", (event, code, details) => this.handleLoadError(details));
-                transProxy.addEventListener("Finished", (event, exit) => {
+        pkTransaction("RefreshCache", [true], {
+                ErrorCode: (code, details) => this.handleLoadError(details),
+
+                Finished: exit => {
                     if (exit == PK_EXIT_ENUM_SUCCESS) {
                         this.setState({timeSinceRefresh: 0});
                         this.loadUpdates();
                     } else {
                         this.setState({state: "loadError"});
                     }
-                });
+                },
+            },
 
-                transProxy.addEventListener("changed", (event, data) => {
-                    if ("Percentage" in data && data.Percentage <= 100)
-                        this.setState({loadPercent: data.Percentage});
-                });
+            notify => {
+                if ("Percentage" in notify && notify.Percentage <= 100)
```

Comment:
Yes, it is. Apparently the `dbus_client.watch()` `notify` callback sends "unwrapped" objects (console.log shows `handleRefresh notify Object { Percentage: 64 }`) while the raw `DBus.Properties.GetAll()` returns the JSON representation of GVariants (something like `Percentage": {"t": "u", "v": "63"}`).